modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/css/main.scss

```diff
@@ -831,6 +831,31 @@ $productMenuWidth: 320px;
 	z-index: 2000;
 }
 
+.dm-field-repeatable-delete-button,
```

Comment:
I'm not sure if we need to use it in system/instance settings. 

If we need to use it in settings maybe makes sense to split it into a new file (and import on `/document_library/css/main.scss` or maybe it's better to wait to use it in settings.

Reply:
Good point! yes, I think we will render this component in two different places, so having a separate css to later import it sounds goods